Sources/nest/Commands/UpdateNestfileCommand.swift

```diff
@@ -14,13 +14,16 @@ struct UpdateNestfileCommand: AsyncParsableCommand {
     @Argument(help: "A nestfile written in yaml.")
     var nestfilePath: String
 
+    @Option(parsing: .upToNextOption, help: "Exclude by repository or version when using reference-only.\n(ex. --excludes owner/repo@0.0.1 owner/repo@0.0.2)")
+    var excludes: [ExcludedTarget] = []
```

Comment:
Sorry, I fixed the issue where multiple `ExcludedTarget`s could not be received: https://github.com/mtj0928/nest/pull/33/commits/406be9a3605f16102325f1817e2308b2c476fc40

I've verified that the following functionality works correctly.

- [x] `$ nest update-nestfile nestfile.yaml --verbose --excludes realm/SwiftLint@0.58.2 realm/SwiftLint@0.59.0`
- [x] `$ nest update-nestfile nestfile.yaml --excludes realm/SwiftLint@0.58.2 realm/SwiftLint@0.59.0 --verbose`
- [x] `$ nest update-nestfile nestfile.yaml --excludes realm/SwiftLint@0.58.2 --verbose`
- [x] `$ nest update-nestfile nestfile.yaml --verbose --excludes realm/SwiftLint@0.58.2`